components/tidb_query/src/rpn_expr/types/expr_builder.rs

```diff
@@ -913,129 +872,4 @@ mod tests {
             .is_ok());
         }
     }
-
-    #[test]
-    #[allow(clippy::iter_skip_next)]
-    fn test_expr_with_val() {
```

Comment:
This test is removed because we use protobuf encoding in metadata. It's not necessary to test the protobuf compatibility with tidb.